README.md

```diff
@@ -21,3 +21,9 @@ If you have an issues specific to the [Realm Mobile Database](https://realm.io/p
 * [Objective-C & Swift](https://github.com/realm/realm-cocoa)
 * [React Native](https://github.com/realm/realm-js)
 * [Xamarin](https://github.com/realm/realm-dotnet)
+
+## Releases
+
+Beta versions will be coming out frequently during the Beta period, and a new relase tag will be added to this repository when it does. You can Watch this repo to get notified when a new Beta version is out.
```

Comment:
This should be "a new release tag will be added to this repository when they do"